dev-libs/ferrisloki/ferrisloki-3.0.13-r3.ebuild

```
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic
+
+DESCRIPTION="Loki C++ library from Modern C++ Design"
+HOMEPAGE="http://www.libferris.com/"
+SRC_URI="mirror://sourceforge/witme/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="static-libs"
+
+RDEPEND=">=dev-libs/libsigc++-2.6:2"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
```

I'm pretty sure `virtual/pkgconfig` should be a BDEPEND for this package.

```suggestion
DEPEND="${RDEPEND}"
BDEPEND="virtual/pkgconfig"
```

---

done.